tequilapi/client/client.go

```diff
@@ -734,3 +736,35 @@ func (client *Client) UpdateTerms(obj contract.TermsRequest) error {
 	defer resp.Body.Close()
 	return nil
 }
+
+// FetchConfig - fetches current config
+func (client *Client) FetchConfig() (map[string]interface{}, error) {
+	resp, err := client.http.Get("config", nil)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+		return nil, fmt.Errorf("fetching config failed with status: %d", resp.StatusCode)
+	}
+
+	bodyBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
I think you can [safely reuse](https://play.golang.org/p/FB9g1BBlxPW) the `parseResponseJson` function we already got no?